charter.html

```diff
@@ -251,14 +262,14 @@ <h2 id=proposals>Proposals</h2>
 
 <h2 id=work-items>Work Items</h2>
 
-<p>A <dfn>Work Item</dfn> is a Proposal that has been formally adopted
-as something the Community Group will work on.  The purpose of adopting
-a Work Item is to develop a specification&mdash;or modifications to
-other specifications&mdash;that enables interoperability between
-different implementations.
-
-<p>Every Work Item has one or more <a href=#editors>Editors</a>, who are
-appointed by the <a href=#chairs>Chairs</a>.
+<p>The Community Group may produce <dfn>Work Items</dfn>—Specifications
+as defined in the
+<a href=https://www.w3.org/community/about/agreements/>Community and
+Business Group Process</a>—a special kind of Community Group Report
+whose purpose is to enable interoperability between independent
```

Comment:
"--a special kind of CGR whose purpose is to enable interop between indep. impl. of the feature it defines." 

I guess this clarity differentiates this CG's "specifications" from other potential types of specifications, but was anyone actually confused about this point?

Reply:
This was my attempt to preserve @martinthomson's change from #19. I'd be happy to tweak this further, if you have a suggestion.